### DISCUSSION TITLE
Grafana (Database) Allow to define schema for postgreql

### DISCUSSION THREAD

**Discussion by mmorel-35** [grafana/grafana#58712] [General]
Hi!

I'm looking for a way to define postgresql's schema in my Grafana configuration.
In the config file, I see this options but there is no way to define the schema for posgresql.
```
type = sqlite3
host = 127.0.0.1:3306
name = grafana
user = root
```

Would it be possible to have this option ?

Thank you

**Comment by dioniseo** [comment]
Hi @mmorel-35 just faced with the same issue. Did you find a way to define it? I was thinking about defining it in the db url using `?currentSchema=` param, but seems like based on https://github.com/grafana/grafana/issues/27435 its not possible.

**Reply by xorima** [reply]
I recently tried with the `?currentSchema` and it's still not possible. 

May be worth digging into the code to understand how this is parsed out OR look into other solutions like dynamic dbs etc. which I know does not work for everyone

**Comment by gelicia** [comment]
Hello, as you may have heard, we are [transitioning away from using discussions to discuss feature requests](https://github.com/grafana/grafana/discussions/73424). Due to the age and number of responses to this discussion, we are deciding to close it. If this is something you would like to see in Grafana, feel free to open an [issue](https://github.com/grafana/grafana/issues) so the discussion can continue. Thank you!
